### PR TITLE
help_url populated for all addons, expanded descriptions

### PR DESCRIPTION
**Pull request by emyoulation**
Updated each .gpr.py file for:
   description
   authors
   authors_email
   audience
   help_url

Version was NOT incremented
This registration files should create addon lists for the Addon Manager that have wiki lists.

Needs to be validated by an experienced developer. This is my 1st Pull Request and I am not confident everything is right. Also, I do not know the process for generating the .tgz files or the json addon list.

### CODE REVIEW

File: source/SuperTool/SuperTool.gpr.py [hunk 1]
```diff
@@ -116,12 +126,15 @@
     name=_("Generic citation filter"),
     description=_("Generic citation filter"),
     version=VERSION,
+    audience = EVERYONE,
     authors=["Kari Kujansuu"],
     authors_email=["kari.kujansuu@gmail.com"],
     gramps_target_version=major_version,
     status=STABLE,
+    audience = EVERYONE,
```
Comment (prculley) [inline source/SuperTool/SuperTool.gpr.py]: Added twice?

File: source/importsources/importsources.gpr.py [hunk 2]
```diff
@@ -33,17 +33,19 @@
...
+         version = '0.0.4',
+         gramps_target_version = major_version,
+         status = STABLE,
+         fname = 'importsources.py',
+         category = "Isotammi",
+         toolclass = 'Tool',
+         optionclass = 'Options',
+         help_url="Addon:Isotammi_addons#Import_CSV_sources
+         tool_modes = [TOOL_MODE_GUI, TOOL_MODE_CLI]
+        )
```
Comment (prculley) [inline source/importsources/importsources.gpr.py:51]: No audience?

File: source/SuperTool/SuperTool.gpr.py (hunk 1 above)
Comment (emyoulation) [reply source/SuperTool/SuperTool.gpr.py]: No, it should have only been once.

Darn! Thought I'd caught all those! It was late and I was tired. 

He had a few with author entries. (Some with "kku")

Some have the licensing header too.

File: source/importsources/importsources.gpr.py (hunk 2 above)
Comment (emyoulation) [reply source/importsources/importsources.gpr.py:51]: Hmm. The resolve button didn't do what I expected.  

You're right, it needs an audience line added too.

**Comment by emyoulation** [comment]
what's the next step?  Do I change it in my repository clone and submit another pull request for those 2? (Be gentle, it is my first Pull Request...)

**Comment by emyoulation** [comment]
@prculley , have I addressed all your comments?  I just discovered that I have to mark each comment as "Resolved" and marked the remaining one.

I think this PR#1 is ready for merging.

**Comment by prculley** [comment]
Looks ok by quick inspection.  I'll build/test and then publish when I get a chance.  Did you build/test?  if you do so on your own machine, you can use the "file:///" syntax in the Gramps addon preferences to point to your local download/listing location.

**Comment by emyoulation** [comment]
I have not gotten to the point of learning the build. :(

But I _have_ pulled down your clone repository to put the gramps52 downloads and listings on a USB drive and set up a "Project" using the file:/// local.

**Comment by prculley** [comment]
Rebuilt and published...

**Pull request merged**